schemas/Flow.js

```diff
@@ -0,0 +1,31 @@
+// The Flow schema represents the configuration and current state of a NiFi Flow
```

Comment:
@nateMJ - This is what we have put together by means for the Flow configuration schema, not all of it will be relevant to initializing NiFi Flows but I want to be sure we're not missing anything either!